bolt-sidecar/src/signer/keystore.rs

```diff
@@ -0,0 +1,108 @@
+//! An ERC-2335 keystore signer.
+
+use std::{
+    ffi::OsString,
+    fmt::Debug,
+    fs,
+    path::{Path, PathBuf},
+};
+
+use alloy::rpc::types::beacon::constants::BLS_PUBLIC_KEY_BYTES_LEN;
+use eyre::eyre;
+
+use lighthouse_bls::Keypair;
+use lighthouse_eth2_keystore::Keystore;
+
+use crate::crypto::bls::BLSSig;
+
+pub const KEYSTORES_DEFAULT_PATH: &str = "keys";
+
+#[derive(Clone)]
+pub struct KeystoreSigner {
+    keypairs: Vec<Keypair>,
+}
+
+impl KeystoreSigner {
+    pub fn new(keys_path: Option<&str>, password: &[u8]) -> eyre::Result<Self> {
```

Comment:
According to the [ERC](https://eips.ethereum.org/EIPS/eip-2335#password-verification) and the [lighthouse implementation](https://github.com/sigp/lighthouse/blob/stable/crypto/eth2_keystore/src/keystore.rs#L250) it is required. It is just a better practice imo, if you want to avoid a password you can always use `password` :)

Reply:
Got it thanks